quickwit/quickwit-indexing/src/actors/indexer.rs

```diff
@@ -878,6 +876,11 @@ mod tests {
                 .delete_opstamp,
             last_delete_opstamp
         );
+        assert!(!universe
+            .quit()
+            .await
+            .into_iter()
+            .any(|s| matches!(s, ActorExitStatus::Panicked)));
```

Comment:
no one letter variable please... It is not that simple to get what this one is about.